src/QA/QA.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -56,7 +56,7 @@ private VacancyReviewSearchResultViewModel MapToViewModel(VacancyReview vacancyR
             return new VacancyReviewSearchResultViewModel
             {
                 IsAssignedToLoggedInUser = vacancyUser.UserId == vacancyReview.ReviewedByUser?.UserId,
-                AssignedTo = isAvailableForReview ? null : vacancyReview.ReviewedByUser.Name,
+                AssignedTo = vacancyReview.ReviewedByUser.Name,
```

Comment:
Is it safe to assume there will always be a ReviewByUser?

Reply:
No it ain't... should null propagate it.

Reply:
`ain't` ain't a real word!